### PR TITLE
Update New-ADServiceAccount  - issue #359

### PR DESCRIPTION
**Pull request by ojrb**
Updated the changes requested for issue #359

### CODE REVIEW

**Comment by ojrb** [comment]
This PR #985 correct the issue #359

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/winserver2012-ps/activedirectory/New-ADServiceAccount.md [hunk 1]
```diff
@@ -403,7 +403,14 @@ Accept wildcard characters: False
 ```
 
 ### -DNSHostName
-Specifies the Domain Name System (DNS) host name.
+Specifies the DNS (Domain Name System) host name of the Service Account.
+
+This parameter sets the value of the FQDN (Fully Quilified Domain Name) for the Service Account created.
```
Comment (nenonix) [inline docset/winserver2012-ps/activedirectory/New-ADServiceAccount.md]:
```suggestion
This parameter sets the value of the FQDN (Fully Qualified Domain Name) for the Service Account created.
```

**Comment by e0i** [comment]
@ojrb Kindly incorporate suggested changes into your PR so that it gets rolled into tech review. Thanks.

**Comment by ojrb** [comment]
@e0i Hi ! Ready the merge of the suggested changes on the PR by @nenonix !

**Comment by ojrb** [comment]
@e0i Update as requested the PowerShell reference for all the commands outside the examples section ! please review and Merge !

**Comment by ojrb** [comment]
@e0i I update the changes requested to the date and time types (Enclosed only in single quotes) and Eliminated the Extra line in the examples as requested! please review and merge

**Comment by ojrb** [comment]
Hi @e0i  I modified the file as requested, eliminating the word "Description" from the examples section, I committed the changes to the PR, please review and Merge !

**Comment by e0i** [comment]
@AndreaBarr This PR that has passed several rounds of review can now be merged. Thank you.

**Pull request merged**